src/common/updating/updating.vcxproj

```diff
@@ -119,7 +119,7 @@
     <Lib>
       <AdditionalLibraryDirectories>
       </AdditionalLibraryDirectories>
-      <AdditionalDependencies>Pathcch.lib;Version.lib</AdditionalDependencies>
+      <AdditionalDependencies>Version.lib</AdditionalDependencies>

```

Comment:
I don't know what `Pathcch.lib` is or why it was added, since Andrey is on vacation, let's move this to its on PR but let's wait before opening it.

Reply:
Opened https://github.com/microsoft/PowerToys/pull/6150